### PR TITLE
Implemented pony name generator for scope default naming

### PR DESCRIPTION
**Pull request by s0larm00n**

### CODE REVIEW

File: drill-common/src/commonMain/kotlin/com/epam/drill/common/PonyNameGenerator.kt [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "Galaxy",
+    "Incandescence",
+    "Lullaby",
+    "Sunshine",
+    "Sweetheart",
+    "Syzygy",
+    "Elision"
+)
+
+fun generatePonyName() = "${firstName[(0 until firstName.size).random()]} ${lastName[(0 until lastName.size).random()]}"
```
Comment (viktorbll) [inline drill-common/src/commonMain/kotlin/com/epam/drill/common/PonyNameGenerator.kt:71]: code duplication

Comment (SergeyPirogov) [reply drill-common/src/commonMain/kotlin/com/epam/drill/common/PonyNameGenerator.kt:71]: Why not to use jfaker?

Comment (s0larm00n) [reply drill-common/src/commonMain/kotlin/com/epam/drill/common/PonyNameGenerator.kt:71]: cuz what we need is generating names of equestria, not some czech republic
it's a long story

Comment (viktorbll) [reply drill-common/src/commonMain/kotlin/com/epam/drill/common/PonyNameGenerator.kt:71]: Anyway, drill-common is not the right place for this thing
At the moment the only possible application for this is the coverage plugin, so it should be put there.

Comment (s0larm00n) [reply drill-common/src/commonMain/kotlin/com/epam/drill/common/PonyNameGenerator.kt:71]: no problem, will fix asap

**Pull request closed without merging**